library/src/morpheus-master/sass/modules/tool/sitemanage/_sitemanage.scss

```diff
@@ -342,17 +342,17 @@
     h3 {
         margin: 0;
         padding: 0;
-		
+
       a {
         border-top: $manageTools-panel-border;
         border-bottom: $manageTools-panel-border;
-        background: $manageTools-heading-background;
+        background: var(--sakai-background-color-2);
         text-decoration: none;
         text-transform: capitalize;
         padding: $standard-spacing;
         display: block;
         margin: 0;
-        
+
         &:hover, &:hover .checkedCount, &:focus, &:focus .checkedCount {
           background: $manageTools-heading-hover-background;
```

Comment:
This need to be updated in library/src/morpheus-master/sass/modules/tool/sitemanage/_sitemanage_variables.scss there's several hard coded colors there